lib/midi_synth/keyboard.ex

```diff
@@ -19,11 +19,11 @@ defmodule MIDISynth.Keyboard do
     iex> MIDISynth.Keyboard.play(synth, 60, 100, 80)
     :ok
   """
-  @spec play(GenServer.server(), Command.note(), Command.duration(), Command.velocity()) :: :ok
-  def play(server, note, duration, velocity \\ 127) do
-    MIDISynth.midi(server, Command.note_on(note, velocity))
+  @spec play(GenServer.server(), Command.note(), Command.duration(), Command.velocity(), Command.channel()) :: :ok
+  def play(server, note, duration, velocity \\ 127, channel \\ 0) do
```

Comment:
Fwiw, I like this order with `channel` being defaulted here. My thought was that people would use `MIDISynth.Keyboard` to check that things worked and maybe for really simple apps, but that it was mostly example code.